elasticsearch/create_concepts_index.json

```diff
@@ -58,13 +58,9 @@
                                 }
                             },
                             "uri": {
-                                "type": "text",
-                                "fields": {
-                                    "keyword": {
-                                        "type": "keyword",
-                                        "ignore_above": 256
-                                    }
-                                }
+                                "type": "object",
+                                "enabled": false,
```

Comment:
Da forsvinner mulighet for å gjøre filtrering på uri ser det ut som. Det brukes på rapporter.